sw/device/silicon_creator/rom_ext/rom_ext_common.ld

```diff
@@ -26,6 +26,22 @@ ENTRY(_rom_ext_start_boot)
 /* DV Log offset (has to be different to other boot stages). */
 _dv_log_offset = 0x10000;
 
+/*
```

Comment:
Would it make sense to pull this, as well as the `.manifest` section declaration below, into a separate linker file fragment that can be INCLUDEd, for reuse purposes.

Reply:
I'm not sure. This would require consistent naming of the `_text_start`, `_text_end`, and `_start_boot` symbols in the common linker files anywhere a manifest is needed. Easy enough if we rename `_rom_ext_start_boot` to just `_start_boot`, but I'm not sure this is a requirement we want to impose.

Reply:
I'll probably rip up our linker scripts in another pass later, so let's leave it as-is.